### DISCUSSION TITLE
Bluetooth audio dows not reach CamillaDSP

### DISCUSSION THREAD

**Discussion by Slaughthammer** [rern/rAudio#1278] [Bug Reports]
I have the issue that while my MPD audio working perfect with CamillaDSP, any audio that comes in via bluetooth disappears somewhere. Running on a RasPi3 v1.2 with an USB soundcard. 
![Screenshot 2023-03-24 194141](https://user-images.githubusercontent.com/97415297/227623672-e057758a-4ddc-4d31-ab9e-fa0f48d02796.png)
![Screenshot 2023-03-24 194858](https://user-images.githubusercontent.com/97415297/227623676-d745eebe-fbb7-4f1d-a608-677440fa51c9.png)
![Screenshot 2023-03-24 223449](https://user-images.githubusercontent.com/97415297/227645799-4d9c19ad-ca2e-434a-9632-be7096ab98b5.png)
Will gladly provide further debug info. when activating bluetooth there is a message saying "playback in FLOAT64LE" which might indicate towards the source of the problem.....

**Comment by NexusPenguin** [comment]
Hi,

Is it normal that you have 6 playback channels ? 

I sometimes have the same problem you have, but that is when I switch from one I2S device to another. 

If you haven't done already, try to turn off Camilla DSP. Do you have sound then ? If so, turn Camilla back on and wait for the error message. If you have that thing about Floart something, try to play with the setting in "Capture device" and/or "Plmayback device". 
Doing so, I usaully get sound again. 

Regards.

**Reply by Slaughthammer** [reply]
6 playback channels for active 3 way stereo speakers, crossover happens in camilla.

When camilla is disabled everything works as expected. Setting the capture device format to anything other than S32LE results in camilla crashing, which is to be expected since the loopback device is set to output S32LE format. My best guess is that the problem is between buezalsa and loopback device. I don't get any error message

**Comment by NexusPenguin** [comment]
Hi,

OK. Have you tried to set the catpure device to something else, like hw:loopback,1

I think it helped when I had my problems. 

If not, then you'll have to wait for someone much more skillend than I am :-(
